### PR TITLE
ESLint: Setup + Documentation (Linting for JS files)

### PR DESCRIPTION
**Pull request by fnormand01**
Same as my previous PR (#20), I put some basic rules and I think that we should all meet to discuss the rules we want to use during the development of this project.

### CODE REVIEW

File: frontend/eslint.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (caleybrock) [inline frontend/eslint.json]: Given that we are defining this config in it's own file, should we remove the config from `package.json`: https://github.com/code-for-canada/project-thundercat/blob/master/frontend/package.json#L19

**Comment by fnormand01** [comment]
I just updated the first step of Pylint and ESLint setup in the documentation. The way to explain it is better this way.

**Comment by MichaelCherry** [comment]
It is setup and working on mine. Very cool

**Comment by caleybrock** [comment]
The first few time I tried this I got: 
```
warning " > eslint-config-airbnb-base@13.1.0" has unmet peer dependency "eslint@^4.19.1 || ^5.3.0".
frontend_1  | warning " > eslint-config-airbnb-base@13.1.0" has unmet peer dependency "eslint-plugin-import@^2.14.0".
```

Seems to be working now, but documenting here incase we need to come back to it.

**Pull request merged**